earth2studio/lexicon/arco.py

```diff
@@ -53,6 +53,9 @@ class ARCOLexicon(metaclass=LexiconType):
         "msl": "mean_sea_level_pressure::",
         "tcwv": "total_column_water_vapour::",
         "tp": "total_precipitation::",
+        "tp06": "total_precipitation_6hr::",
+        "z": "geopotential_at_surface::",
```

Comment:
is this a ecmwf standard? what is it labels on CDS / parameter db?

Reply:
tp06 is ECMWF standard https://codes.ecmwf.int/grib/param-db/260267
z is a bit more tricky - since it can refer to both pressure levels or surface (which can also be called orography) https://codes.ecmwf.int/grib/param-db/129

Reply:
I switched to WB2 for my tests.